src/calculateRentalCost.js

```diff
@@ -4,7 +4,28 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalPrice = 0;
+  const cost = 40;
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+  const longRental = 7;
+  const mediumRental = 3;
+
+  if (days >= longRental) {
+    totalPrice += cost * days - longTermDiscount;
```

Comment:
The logic for applying the long-term discount is correct, but ensure that the discount is only applied once. The current logic correctly applies the discount if the rental is 7 days or more.